README.md

```diff
@@ -48,12 +48,70 @@ A quick guide to installing, configuring, and running an instance of the SDL Cor
 
 ####Windows
 
-A quick guide to configuring and running an instance of the SDL Core on a Windows OS.
-For details [see](How-to-build-and-run-SDL-on-windows)
+> Required OS: Windows 7 64-bits
+
+##### Common steps for Windows and Qt platforms
+
+* Download and install [Cmake](https://cmake.org/files/v3.2/cmake-3.2.3-win32-x86.exe)<br>
+Note: it will ask you about adding to the `PATH` environment variable. Press `Yes`.
+* Download and install [python](https://www.python.org/ftp/python/2.7.10/python-2.7.10.msi) and add it to the `PATH`
+environment variable
+* Download and install [perl](http://www.activestate.com/activeperl/downloads/thank-you?dl=http://downloads.activestate.com/ActivePerl/releases/5.20.2.2002/ActivePerl-5.20.2.2002-MSWin32-x86-64int-299195.msi). Add it to `PATH`
+* Download and install [git](https://git-scm.com/download/win)
+* Download and install [GStreamer SDK](http://docs.gstreamer.com/display/GstSDK/Installing+on+Windows). You will need both Runtime and Development files. Add path to **bin** folder to `PATH`. Create new environment variable SDL_GSTREAMER_DIR with path to SDK (e.g. `D:\gstreamer-sdk\0.10\x86_64`).
+
+##### Steps for Windows platform
+* Download [sqlite3 sources](https://www.sqlite.org/2015/sqlite-amalgamation-3090200.zip) and [sqlite3 binaries](https://www.sqlite.org/2015/sqlite-dll-win64-x64-3090200.zip). Extract all files to the same directory. Create new  environment variable `SDL_SQLITE_DIR` and assign path where sqlite sources and binaries were extracted to this variable.
+* Install [Windows SDK 7.1](https://www.microsoft.com/en-us/download/details.aspx?id=8442)
+* Install [Visual Studio 2013](https://www.microsoft.com/en-us/download/confirmation.aspx?id=44914)
+
+##### Steps for Qt platform
+* Install [Visual Studio 2010](http://go.microsoft.com/?linkid=9709969)
+* Then install [Visual Studio SP1](https://www.microsoft.com/en-us/download/confirmation.aspx?id=23691)
+* Install [Qt framework](http://download.qt.io/official_releases/online_installers/qt-unified-windows-x86-online.exe).
```

Comment:
@OHerasym, maybe better to provide the direct link from http://download.qt.io/archive/qt/5.5/5.5.1/ ?